commonutils/src/main/java/io/intrepid/commonutils/StringUtils.java

```diff
@@ -44,6 +41,69 @@
         }
     };
 
+    // region TextUtils methods
+    // These methods are copied from TextUtils class so that we don't need to mock them when running unit tests
+
+    /**
+     * Returns true if the string is null or 0-length.
+     *
+     * @param str the string to be examined
+     * @return true if str is null or zero length
+     */
+    public static boolean isEmpty(@Nullable CharSequence str) {
```

Comment:
why would they need to mock it out if they were using the TextUtils version? Wouldn't TextUtils simply be a dependency of this library?